plugins/modules/aci_vmm_enhanced_lag_policy.py

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Dev Sinha (@DevSinha13) <devsinh@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_vmm_enhanced_lag_policy
+short_description: Manage Enhanced LACP Policy for Virtual Machine Manager (VMM) in Cisco ACI (lacp:EnhancedLagPol)
+description:
+- Manage Enhanced LACP Policy for VMM domains on Cisco ACI fabrics.
+- The Enhanced LACP Policy allows you to configure advanced Link Aggregation Control Protocol (LACP) settings for virtual switches in VMM domains.
+- This policy is a child of the C(vmmVSwitchPolicyCont) class.
+
+options:
+  name:
+    description:
+    - The name of the Enhanced LACP Policy.
+    type: str
+  domain:
+    description:
+    - The name of the virtual domain profile where the Enhanced LACP Policy is applied.
+    type: str
+    aliases: [ domain_name, domain_profile ]
+  vm_provider:
+    description:
+    - The virtualization platform provider for the VMM domain.
+    type: str
+    choices: [ cloudfoundry, kubernetes, microsoft, openshift, openstack, redhat, vmware ]
+  lacp_mode:
+    description:
+    - The LACP mode for the policy.
+    - Determines whether the policy initiates or responds to LACP negotiations.
+    type: str
+    choices: [ active, passive ]
+    default: active
```

Comment:
your change causes sanity failures, we document the apic default behaviour in the description only
```suggestion
  lacp_mode:
    description:
    - The LACP mode for the policy.
    - Determines whether the policy initiates or responds to LACP negotiations.
    - The APIC defaults to C(active) when unset during creation.
    type: str
    choices: [ active, passive ]
```